uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/my-leave/apis/entitlement-api_test.js

```diff
@@ -0,0 +1,55 @@
+define([
+  'leave-absences/shared/modules/apis',
+  'leave-absences/shared/apis/entitlement-api',
+], function() {
+  'use strict';
+
+  describe('EntitlementAPI', function (){
+    var entitlementAPI;
+
+    beforeEach(module('leave-absences.apis'));
+
+    beforeEach( inject( function(_EntitlementAPI_) {
+      entitlementAPI = _EntitlementAPI_;
+    }));
+
+    it('has expected end points', function() {
+      expect(Object.keys(entitlementAPI)).toEqual(['all', 'breakdown']);
+    });
+
+    describe('gets all entitlements', function() {
+      it('response contains period id', function() {
+        entitlementAPI.all().then(function(response) {
+          expect(response).toEqual(jasmine.objectContaining({
+            "period_id": "1"
+          }));
+        });
+      });
+
+      it('gets all entitlements with remainder/balance', function() {
+        var all_entitlements =
```

Comment:
I don't know what you are doing here, but this can't be right. Why are you splitting the line right after the `=` ? Why not simply
```js
EntitlementAPI.all("", true).then(function(allEntitlements) {
  expect(allEntitlements).toEqual(jasmine.objectContaining({
    'remainder': {
      'current': 30,
      'future': 17
    }
  }));
```